pkgs/development/libraries/tclap-1.4/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchgit
+, cmake
+, doxygen
+, python3
+}:
+stdenv.mkDerivation {
+  pname = "tclap";
+  version = "1.4-3feeb7b";
```

Comment:
```suggestion
  version = "1.4.0-rc1";
```

Reply:
The official `1.4.0-rc1` tag doesn't have unit tests and is missing pkgconfig files. That is why I went with this slightly more recent version. If I use 1.4.0-rc1 I will have to apply patches similar to what this commit is providing. See here: https://github.com/mirror/tclap/compare/1.4.0-rc1..1.4

I will add a comment to that effect.